### ISSUE TITLE
repo reorganization

### ISSUE THREAD

**Issue by anandology** [datameet/covid-19-indian-state-reports#1]
I think it is a good idea to extract the data from the archived pdfs and publish in this repo it self. It means we also need to keep the code. 

I suggest the following layout:

```
archive/
    ap/
        file1.pdf
        file2.pdf
    ka/
    kl/
    ..
data/
  ap/
  ka/
  kl/
covid19/
    __init__.py
    download.py
    extract.py
```

**Comment by anandology** [comment]
@aadityadar what do you think?

**Comment by aadityadar** [comment]
of course, please go ahead! btw, what do you mean by 'data'? does that mean the parsed data? the reorganization looks good to me.

**Comment by anandology** [comment]
On Wed, Apr 8, 2020 at 10:35 PM Aaditya Dar <notifications@github.com>
wrote:

> of course, please go ahead! btw, what do you mean by 'data'? does that
> mean the parsed data? the reorganization looks good to me.
>

Thanks. Will do that.

Also, I'm thinking that it may be a good idea to archive the files with the
original name. What do you think?

Could you please give me permission to access this repo? I'm already part
of datameet, but I don't think I have write access to this repo.

**Comment by thejeshgn** [comment]
@anandology given.

**Comment by thejeshgn** [comment]
@anandology reorg looks good

**Comment by anandology** [comment]
Thanks! I've started pushing the new files to `archive/`. Will move the old ones slowly. 

I'll keep the issue open until all the files are moved.

**Comment by Cosmopal** [comment]
Is parsed data also going to be available? By parsed, I mean text data. At least of the ones that are in English
